services/auths.js

```diff
@@ -1,34 +1,27 @@
 const users = require('../data/dbModels/users');
-const organizations = require('../data/dbModels/organizations');
 const axios = require('axios');
 const jwt = require('jsonwebtoken');
 const { secret, slack } = require('../config');
 
 async function loginUser(accessToken, userId) {
   let user = await users.readBySlackId(userId);
+  if (!user || !user.slack_mem_id) {
+    return {
+      statusCode: 401,
+      data: {
+        message: 'Invalid login credentials',
```

Comment:
@erozonachi Would this be an appropriate response message for the user? Given that we are using slack authentication and the user's login credentials have already been okayed by slack.